prototype2/common/EFUArgs.cpp

```diff
@@ -140,7 +165,8 @@ EFUArgs::Status EFUArgs::parseFirstPass(const int argc, char *argv[]) {
   } catch (const CLI::ParseError &e) {
     CLIParser.exit(e);
   }
-  if ((*HelpOption and not *DetectorOption) or (not *HelpOption and not *DetectorOption)) {
```

Comment:
This logic can be simplified - but I would guess not to the intended behaviour

Reply:
This is not new logic, it has only been reformatted using clang-format. Regardless; when investigating the truth table of this if statement, the logic appears to be correct.

Reply:
I know, I just noticed it because of the change in formatting, too bad, but you wrote 
it anyway.

The point is that HelpOption is useless in this logic statement, the result only depends on DetectorOption in which case I would simplify it to 
if (not *DetectorOption)